modules/xxhash/0.8.2/MODULE.bazel

```diff
@@ -0,0 +1,5 @@
+module(
+    name = "xxhash",
+    version = "0.8.2",
+    repo_name = "com_github_cyan4973_xxhash",
```

Comment:
Similar as the other PR, we probably don't need `repo_name = "com_github_cyan4973_xxhash",`

Reply:
How do you ensure naming conflict with the same name and two different repo_name ?
It's theoric question. I have no example in mind yet